viewing_party/party.py

```diff
@@ -1,23 +1,131 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movie = {
+        "title": title, 
+        "genre": genre,
+        "rating": rating
+    }
+
+    if None in movie.values():
+        return None
+
+    else:
+        return movie
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, movie_title):
+    for movie in list(user_data["watchlist"]):

```

Comment:
Nice way to avoid the modifying while iterating situation. This has the benefit that it would safely move all movies that match the title we're looking for.